doc/releases/release-notes-2.2.rst

```diff
@@ -27,6 +27,12 @@ Deprecated in this release
 Stable API changes in this release
 ==================================
 
+* USB
+
+  * Users are obligated to enable USB by themselves.
```

Comment:
I will give a shot at another description here @dbkinder and @emob-nordic:
------
* The `usb_enable()` function, which was previously invoked automatically by the USB stack, now needs to be explicitly called by the application in order to enable the USB subsystem.
* The `usb_enable()` function now takes a parameter,  `usb_dc_status_callback` which can be set by the application to a callback to receive status events from the USB stack. The parameter can also be set to `NULL` if no callback is required.

----

Reply:
lovely - I like it. IMHO this is how it should be